eitprocessing/features/breath_detection.py

```diff
@@ -76,13 +89,26 @@ def find_breaths(self, sequence: Sequence, continuousdata_label: str) -> Interva
         expiration, and end of expiration.
 
         Args:
-            sequence: the sequence that contains the data
-            continuousdata_label: label of the continuous data to apply the algorithm to
+            container: a ContinuousData object that contains the data or a Sequence object
+                that that contains the continuous data
+            continuousdata_label: optional, label of the continuous data contained in the sequence
 
         Returns:
             A list of Breath objects.
         """
+        msg = f"`find_breaths()` expects a Sequence or ContinuousData object as first argument, not {type(container)}."
+        raise NotImplementedError(msg)
```

Comment:
I think it should be a `TypeError`

`NotImplementedError` is mainly in case things are still in development, rather than if they are not intended to be implemented at all (see [the first note here](https://docs.python.org/3/library/exceptions.html#NotImplementedError))

Reply:
I think in the future we will expand this to support `EITData` objects as well. But `TypeError` fits better for now. Fixed in 8fc694ddc56fdb90b88e26bcf52051be043eac9a.

Reply:
In that case, you can still throw a `NotImplementedError` specifically for `EITData` objects. But as it was written, it was throwing it for any type that wasn't supported.